### ISSUE TITLE
Migrating from Quiver

### ISSUE THREAD

**Issue by patrickf3139** [BoostIO/Boostnote#2566]
Hi there, I was a long time Evernote user. A month ago, I found Quiver and despite its lack of recent development, migrated onto it. Then, on the off chance of looking up some Quiver news, I found Boostnote and absolutely fell in love with the project. It's 90% what I need! It may not be 100% but I'd say Quiver was only 70% and Evernote 50% what I need.

Anyway, I immediately tried migrating from Quiver but ran into big roadblocks. Firstly, the only format that Boostnote can import that Quiver can export is Markdown. Unfortunately, exporting Markdown with Quiver has the following problems
1. Small bugs that I can fix by hand but multiplied by 1000s of notes, I realized it would take me a whole day to fix all of them should I go that route
2. No tags exported. I have tagged most of my notes and it'd be difficult to retag everything exactly
3. No images or attachments. Not a big deal but I do have about 100 notes with images or PDF

I realized the Quiver -> Evernote -> Boostnote route would work, but that would require me to migrate my 1 month of many changes back onto Evernote. And after much investigation, I realized that would have to happen by hand, and that'd take me days.

So...now I'm really sad and wondering if anyone, please anyone, has written a script to migrate Quiver to Boostnote, or has any ideas

TL;DR Could someone give me ideas or a script to migrate from Quiver to Evernote?
